src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -126,7 +126,12 @@ static ExperimentalFeature()
                     description: "Support the null coalescing operator and null coalescing assignment operator in PowerShell language"),
                 new ExperimentalFeature(
                     name: "PSPipelineChainOperators",
-                    description: "Allow use of && and || as operators between pipeline invocations"),
+                    description: "Allow use of && and || as operators between pipeline invocations")
+#if !UNIX
+                ,new ExperimentalFeature(
```

Comment:
C# allows for the trailing comma, so you should put it back on line 129

Reply:
Fixed.